### DISCUSSION TITLE
VSync

### DISCUSSION THREAD

**Discussion by WalmartWhiteCat** [ppy/osu#25356] [Q&A] [answered]
Okay, so why is osu!Lazer randomly forcing VSync now? I have refresh rate set to 4x. I know it's registering these changes since my delay changes when I change the rate limiter. I also know it's not just me since my friend is also experiencing this. I'm just confused, I'm fine with it as long as it doesn't mess with my delay.

Alright so nevermind, I guess. I found out why. Damn, this is weird, I never thought a renderer would force VSync but I guess you learn something new every day. Sorry for the useless question whoever is reading this.

**Comment by bdach** [comment]
> Okay, so why is osu!Lazer randomly forcing VSync now?

It's not? Never should be?

> Damn, this is weird, I never thought a renderer would force VSync

What does this mean?

**Reply by peppy** [reply]
I think they found that they forced vaync in their drivers.

**Reply by WalmartWhiteCat** [reply]
For some reason the default renderer in Lazer forces VSync for me. OpenGL doesn't do this so I was confused. Now I just use OpenGL

**Reply by bdach** [reply]
@WalmartWhiteCat what platform are you playing on? also fullscreen or borderless/windowed?

**Reply by WalmartWhiteCat** [reply]
I am playing on Windows PC. I play borderless

**Reply by bdach** [reply, marked as answer]
Is fullscreen any better?

**Reply by WalmartWhiteCat** [reply]
I can check, just a minute

Edit: Yes, fullscreen completely fixes the issue. It's still a little upsetting though, as I really like borderless for switching between screens quickly. Though, it doesn't really matter since there are two renderers. I'm thankful that Peppy or whoever suggested multiple renderers has the foresight they do.
